src/main/resources/query.sql

```diff
@@ -3,6 +3,7 @@ CREATE TABLE login
   id serial NOT NULL,
   username character varying NOT NULL,
   password character varying NOT NULL,
+  granted_role varying NOT NULL,
```

Comment:
why not character varying. The format is static. 
Consistency is better, I believe